src/api/handlers/TutorUpdateHandler.ts

```diff
@@ -0,0 +1,23 @@
+import { Handler } from "aws-lambda";
+import { TutorService } from "../services/TutorService";
+
+class TutorUpdateHandler {
+    private service: TutorService;
+    
+    constructor() {
+        this.service = new TutorService();
+    }
+
+    public execute(event: any) {
+        const tutor = this.service.update(JSON.parse(event.pathParameters.id), JSON.parse(event.body));
```

Comment:
@Aristevao here you don't need to use `JSON.parse()` with `event.pathParameters.id`, because `id` is not an object.